docs/eventing/channels/channel-types-defaults.md

```diff
@@ -6,15 +6,37 @@ showlandingtoc: "true"
 ---
 
 # Channel types and defaults
+Knative uses two types of channels:
 
-Knative provides the InMemoryChannel channel implementation by default. This default implementation is useful for developers who do not want to configure a specific implementation type, such as Apache Kafka or NATSS channels.
 
-**NOTE:** InMemoryChannel channels should not be used in production environments.
+* The generic Channel object
+* Channel implementations that each have their own custom resource definitions (CRDs), such as
+`InMemoryChannel` and `KafkaChannel`.
 
-The default channel implementation is specified in the `default-ch-webhook` ConfigMap in the `knative-eventing` namespace.
-For more information about modifying ConfigMaps, see [Configuring the Eventing Operator custom resource](../../../docs/install/operator/configuring-eventing-cr).
+The custom Channel implementations each have their own event delivery mechanisms, such as in-memory
+or broker-based. Example brokers include Kafka and GCP Pub/Sub.
 
-In the following example, the cluster default channel implementation is InMemoryChannel, while the namespace default channel implementation for the `example-namespace` is KafkaChannel.
+Knative provides the `InMemoryChannel` channel implementation by default.
+This default implementation is useful for developers who do not want to configure a specific
+implementation type, such as Apache Kafka or NATSS channels.
+
+Through the generic Channel object you can create a channel without specifying which channel
+implementation CRD is used.
+This is useful if you do not care about the properties a particular channel provides, such as
+ordering and persistence, and you want to use the implementation selected by the operator.
+The operator controls the default settings through the `default-ch-webhook` ConfigMap in the
+`knative-eventing` namespace.
```

Comment:
Use "cluster administrator" consistently instead of "operator"? (assuming that's what's meant here)
I think it's confusing for us to use operator since it's an overloaded term with the [Knative Operator](https://github.com/knative/operator)

Reply:
Good idea, I'll do so.